### PR TITLE
I've corrected the typos, added missing punctuation, and adjusted formatting for better readability.

### PR DESCRIPTION
**Pull request by vwake01**

### CODE REVIEW

**Comment by nobu** [comment]
Please fold long lines.

**Review by nobu — CHANGES_REQUESTED** [review]
Please fold long lines, and remove trailing spaces at end of lines.

**Comment by nobu** [comment]
Also the irb document is not managed here.

**Comment by nobu** [comment]
* Enclosing in back qoutes is code in Markdown format, not a typo of single quotes.
* Doubly enclosing in brackets is for Markdown reference, gathered at the end of this file.

File: NEWS.md [hunk 1]
```diff
@@ -7,22 +7,19 @@ Note that each entry is kept to a minimum, see links for details.
...
-  as if they were frozen. If they are mutated a deprecation warning is emited.
-  These warnings can be enabled with `-W:deprecated` or by setting `Warning[:deprecated] = true`.
-  To disable this change you can run Ruby with the `--disable-frozen-string-literal` command line
-  argument. [[Feature #20205]]
+*String literals in files without a frozen_string_literal comment now behave as if they were frozen.
+If they are mutated a deprecation warning is emitted. These warnings can be enabled with -W:deprecated or by setting Warning[:deprecated] = true. 
+To disable this change, you can run Ruby with the --disable-frozen-string-literal command line argument. [Feature #20205]
 
-* `it` is added to reference a block parameter. [[Feature #18980]]
+*A new syntax **'it': value is added to reference a block parameter. [Feature #18980]
```
Comment (nobu) [inline NEWS.md:14]: I can't get your intention.
What does `value` mean here?

Comment (vwake01) [reply NEWS.md:14]: it make it easy to read as it show that 'it' is a syntex

Comment (nobu) [reply NEWS.md:14]: It is the part "A new syntax", right?
I'm not sure where/how the sentence is separated.

`A new syntax **'it'` and `value is added ...`?
Or `A new syntax` means `**'it': value`?
The latter does not make sense obviously, but then what does `value` mean in the former?

**Comment by hsbt** [comment]
Unfotunately, there is no improvement for us.

**Comment by nobu** [comment]
These look fine to me.

* "emited" → "emitted"
* The comma after "To disable this change"
* "similar" → "similarly"

**Pull request closed without merging**